### PR TITLE
05 sep

### PR DESCRIPTION
**Pull request by Kapil0187**

### CODE REVIEW

File: app/views/home/show.html.erb [hunk 1]
```diff
@@ -13,27 +13,27 @@
...
-	    		<th scope="col"> Bus Number </th>
-	    		<th scope="col"> Bus Name </th>
-	    		<th scope="col"> Total Seats </th>
-				<th scope="col"> Remaning seats </th>
-	    		<th scope="col"> Departure Time </th>
-	    		<th scope="col"> Arrival Time </th>
-	    		<th scope="col"> Ticket Price </th>
-	    		<th scope="col"> Bus Booking </th>
-	  		</tr>
+	  			<tr>
```
Comment (aadi163) [inline app/views/home/show.html.erb:16]: Make sure you have to give 2 spaces.

File: app/views/home/show.html.erb [hunk 2]
```diff
@@ -13,27 +13,27 @@
...
-	       		<td> <%= bus.name %> </td>
-	       		<td> <%= bus.total_seets %> </td>
-		   		<td> <%= bus.remaning_seats %> </td>
-	       		<td> <%= bus.departure_time.to_fs(:time) %> </td>
-	       		<td> <%= bus.arrival_time.to_fs(:time) %> </td>
-	       		<td> <%= bus.price %> </td>
+	       			<td> <%= bus.id %> </td>
+	       			<td> <%= bus.name %> </td>
+	       			<td> <%= bus.total_seets %> </td>
+		   			<td> <%= bus.remaning_seats %> </td>
```
Comment (aadi163) [inline app/views/home/show.html.erb:33]: Here also you are giving more spaces

File: app/models/train.rb [hunk 3]
```diff
@@ -1,4 +1,12 @@
 class Train < ApplicationRecord
+  validates :name, presence: true
+  validates :total_seets, presence: true
+  validates :departure_time, presence: true
+  validates :arrival_time, presence: true
+  validates :price, presence: true
+  validates :direction_id, presence: true
```
Comment (aadi163) [inline app/models/train.rb:7]: Why did you write validations again and again?  you can write validations in single line. If all this columns are in a same table.

Comment (Kapil0187) [reply app/models/train.rb:7]: okay

**Pull request merged**